modules/df/df_tools/df_tools_media/modules/df_tools_gallery/config/install/core.entity_form_display.entity_gallery.media.default.yml

```diff
@@ -24,17 +24,12 @@ content:
     settings: {  }
     third_party_settings: {  }
   entity_gallery_media:
-    type: entity_browser_entity_reference
-    weight: 5
+    type: entity_reference_autocomplete
```

Comment:
This actually seems off too. Sorry to miss it. Did it really remove the entity browser usage here? That is definitely wrong.